pkg/kubelet/apis/config/types.go

```diff
@@ -374,12 +374,12 @@ type KubeletConfiguration struct {
 	// A set of ResourceName=ResourceQuantity (e.g. cpu=200m,memory=150G,ephemeral-storage=1G,pid=100) pairs
 	// that describe resources reserved for non-kubernetes components.
 	// Currently only cpu, memory and local ephemeral storage for root file system are supported.
-	// See http://kubernetes.io/docs/user-guide/compute-resources for more detail.
+	// See https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ for more detail.
```

Comment:
Would https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources be more appropriate here?

Reply:
Right